src/tests/lib/qunit_config.js

```diff
@@ -1 +1,41 @@
-QUnit.config.testTimeout = 6400;
+/* globals badger:false */
```

Comment:
I'm (slowly) moving the third-party stuff into `lib/vendor`. I was thinking `lib` (vs. `js`) could be our own library/non-main-entry-point type files.